_startup/camino.md

```diff
@@ -0,0 +1,33 @@
+Sur le terrain, les citoyens et les élus ne découvrent les projets qui les concernent qu’au moment d’une consultation ou d’une enquête publique réglementaire faute d’accès simple et durable aux informations sur les projets miniers. Pris de cours, face à des délais très courts et des dossiers complexes ou volumineux, ils dénoncent un manque de considération et de transparence. Une telle situation ne permet pas d’engager des débats ouverts, éclairés et sereins sur les projets miniers en France.
+
+**Camino, le *cadastre minier numérique ouvert*, ambitionne d'ouvrir et simplifier l’administration des mines.**
+Avec Camino, chacun pourra connaître l'existence et les caractéristiques des titres miniers et suivre les projets en cours sur une carte interactive.
+Basé sur un référentiel commun ouvert et partagé des dossiers miniers et des autorisations administratives, Camino va résoudre les difficultés des entreprises, des citoyens et des agents publics.
+
+* Les entreprises minières pourront connaitre l’état du domaine minier puis déposer en ligne leurs demandes, les rapports et les données réglementaires. Elles pourront aussi visualiser l’avancement des instructions, suivre leur portefeuille de titres et répondre simplement à leurs obligations à chaque étape des projets.
+* Les agents publics pourront simplement accéder aux mêmes dossiers et données partagés entre les services des administrations et opérateurs publics impliqués. Il sera possible de co-instruire les demandes, de suivre et anticiper chaque moment de vie d’un projet, de trouver et valoriser les données sur le domaine minier national simplement et sans intermédiaire.
+* Les ONG, les citoyens et leurs élus pourront consulter à tout moment les dossiers et les données publiques, à l’échelle de leur territoire pour mieux contribuer, sans précipitation, aux procédures de participation du public et suivre de manière transparente l’actualité des projets.
+
+**Notre premier pas sur Camino**
+Une première preuve de concept constituée de tableaux couplés à une carte interactive valide la possibilité de constituer un référentiel commun des titres et projets miniers. Maintenant, il s’agit de l’étendre à l’ensemble des titres valides énergétiques, non énergétiques et les stockages souterrains (hors substances utiles à l’énergie atomique) pour présenter une cartographie complète et à jour du domaine minier et des projets en cours.
+Nous dématérialiserons les premières procédures les plus fréquentes du code minier et proposerons un tableau de bord aux entreprises qui expérimenteront le service Camino.
+Avec les données recueillies, nous automatiserons l’actualisation du cadastre minier et nous diffuserons dans un standard électronique ouvert les données et documents publics déjà disponibles d’ici six mois.
```

Comment:
sur data.gouv.fr non ? Dans ce cas le dire explicitement.

Reply:
Of course et ça va mieux en le disant. On le mettra aussi sur nos sites métiers, mais on va pas faire une liste à rallonge.